TheWidlarzGroup/react-native-video
# feat(android): support hiding Exoplayer video duration on android

## Summary
This PR adds support for hiding the duration on Android devices.

### Changes
  -  Implemented a new prop to hide the duration on Android.
  -  Updated relevant Android-specific components to respect the new prop.

## Test plan
    1. Apply the new prop hideDuration = true to hide the duration.
    2. Verify that the video player does not display the duration when the prop is true.
